app/Http/Controllers/Web/ImagesController.php

```diff
@@ -63,9 +63,9 @@ public function show(string $hash, Request $request)
         $response->headers->set('Surrogate-Control', 'max-age=31536000');
         $response->headers->set('Surrogate-Key', 'post-'.$post->id);
 
-        // But we want browsers to always re-validate before serving this image from local
-        // cache, so that image rotations/deletions are respected immediately:
-        $response->headers->set('Cache-Control', 'no-cache, public');
+        // And we want browsers to cache but re-validate with us before returning cached content
+        // on a subsequent load, so that image rotations/deletions are respected immediately:
+        $response->headers->set('Cache-Control', 'max-age=31536000, must-revalidate, no-cache, public');
```

Comment:
With this does it no longer add the `private` to the end?

Reply:
That part was fixed in #940, although I didn't dig into why at the time (which is what it seems like that snippet does). Symfony seems to require Responses to always have either `private` or `public` on them (and if neither is set, it'll generally mark it as `private` by default).

This is also accepted by Symfony without changes, and seems to be accepted by Fastly as well (whereas before, even though the `Cache-Control` and `Surrogate-Control` looked correct to me, they were being discarded by Fastly when run in production).